openqa-qam-sle

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env bash
+
+# Search openQA jobs related to a given update inside job groups:
+# - Maintenance: Single Incidents / Core Incidents
+# - Maintenance: Aggregated updates / Core Maintenance Updates
+# Default search within the last 5 days for "Aggregated updates" builds
+# Usage: eg
+#   openqa-qam-sle -a SUSE:Maintenance:34576:337335
+#   openqa-qam-sle -a SUSE:Maintenance:34577:337331
+#   openqa-qam-sle -a SUSE:Maintenance:34564:337279
+#   openqa-qam-sle -a SUSE:Maintenance:34581:337626
+
+set -e
+
+# declare -a groups=(
+#     233 # 15-SP1
+#     306 # 15-SP2
+#     367 # 15-SP3
+#     439 # 15-SP4
+#     490 # 15-SP5
+#     546 # 15-SP6
+#     106 # 12-SP3
+#     282 # 12-SP5
+#     521 # 15-SP4-TERADATA
+#     191 # 12-SP3-TERADATA
+# )
+
+# Maintenance: Single Incidents / Core Incidents
+declare -A dict_group=()
+dict_group["15-SP1"]="233"
+dict_group["15-SP2"]="306"
+dict_group["15-SP3"]="367"
+dict_group["15-SP4"]="439"
+dict_group["15-SP5"]="490"
+dict_group["15-SP6"]="546"
+dict_group["12-SP3"]="106"
+dict_group["12-SP5"]="282"
+dict_group["15-SP4-TERADATA"]="521"
+dict_group["12-SP3-TERADATA"]="191"
```

Comment:
You can declare an associative array in bash like this: https://github.com/Martchus/PKGBUILDs/blob/master/devel/versions.sh

So there's no need for all the quotes and repetition of `dict_group`.

Reply:
In general I prefer to quote the strings to avoid some sporadic side effects, like blank and so no.

Reply:
You can at least avoid the repetition of `dict_group` and the quotes on the right hand side.